nibabel/streamlines/tests/test_tractogram.py

```diff
@@ -770,7 +785,7 @@ def test_tractogram_extend(self):
                              (extender, True)):
             first_arg = t.copy()
             new_t = op(first_arg, t)
-            assert_equal(new_t is first_arg, in_place)
+            assert (new_t is first_arg) is in_place
```

Comment:
This one is fine as is, but when comparing booleans, I tend to use `is` for literals, and `==` for variables that are boolean. `is` feels like it's doing something clever, so it makes me stop and check everything for correctness, which isn't something you want to do every time you read a test.

```suggestion
            assert (new_t is first_arg) == in_place
```